docker-compose-addons.yml

```diff
@@ -100,11 +100,11 @@ services:
       PG_DB_PASS: ${PG_DB_PASS}
 
       PROJECT_ID: ${GCP_PROJECT_ID}
-      GOOGLE_APPLICATION_CREDENTIALS: ${GOOGLE_APPLICATION_CREDENTIALS}
+      GOOGLE_APPLICATION_CREDENTIALS: /home/gcp_key.json
 
       LOGGING_LEVEL: ${ISS_LOGGING_LEVEL}
     volumes:
-      - ${GOOGLE_APPLICATION_CREDENTIALS}:/google/gcp_credentials.json
+      - ./resources/google/${GOOGLE_ACCESS_KEY_NAME}:/home/gcp_key.json
```

Comment:
To clarify the changes made to the GOOGLE_APPLICATION_CREDENTIALS and mounted volumes, this is to ensure the security key is mounted and accessed the same way across all of the docker-compose files. The CV Manager API already mounts the environment variable in this manner and it is good for this to be consistent.